src/Tasks/Microsoft.NET.Build.Tasks/JoinItems.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Linq;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public class JoinItems : TaskBase
```

Comment:
Yeah I would like to see the impact on the build, I'm only testing 1.1 SDK - so I have no idea what impact 2.0 SDK has on perf.

Is there a good write up on what we're trying to solve with all these "fixups"? They smell like we're missing feature asks.